docs/components/CurrentCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactElement } from "react";
+import { useMicroSDeckContext } from "../../lib/src";
+import React from "react";
+
+export function CurrentCard(): ReactElement {
+	const { currentCardAndGames } = useMicroSDeckContext();
+
+	if (!currentCardAndGames) {
+		return (<>Unfortunately, This little trick only works while you have a MicroSD card inserted. Try sticking one in</>);
+	}
+
+	const [card, games] = currentCardAndGames;
+
+	return (
+		<>
+			<br/>
+
+			You current have card "<b>{card.name}</b>" inserted.<br/>
+			But did you know it also has a unique ID?<br />
+			Here it is: <b>{card.uid}</b>
+			<br />
+			<br />
+			For good measure, Here are all the game Id's too:<br />
+			{
+				games.map(v => (
+					<>
+						<b>{v.name}:</b> {v.uid}<br />
+					</>
+				))
+			}<br />
```

Comment:
The user is prompted to insert a MicroSD when they don't have one, but they don't get prompted to install a game to an empty card in this flow. Might be worth prompting that edge case too